### ISSUE TITLE
Gem bigdecimal-1.4.0 install error

### ISSUE THREAD

**Issue by Sega100500** [ruby/bigdecimal#121]
Linux Debian 8
Apache 2.4
Ruby 2.5.3, 2.3.8

During an installation bigdecimal-1.4.0 running by 'gem update' is occurred error:

```
Updating installed gems
Updating bigdecimal
Fetching bigdecimal-1.4.0.gem
Building native extensions. This could take a while...
ERROR:  Error installing bigdecimal:
        ERROR: Failed to build gem native extension.

    current directory: /home/ruby/.rbenv/versions/2.3.8/lib/ruby/gems/2.3.0/gems/bigdecimal-1.4.0/ext/bigdecimal
/home/ruby/.rbenv/versions/2.3.8/bin/ruby -I /home/ruby/.rbenv/versions/2.3.8/lib/ruby/site_ruby/2.3.0 -r ./siteconf20181224-26637-iv5fbs.rb extconf.rb
checking for labs() in stdlib.h... yes
checking for llabs() in stdlib.h... yes
checking for finite() in math.h... yes
checking for isfinite() in math.h... no
checking for struct RRational in ruby.h... no
checking for rb_rational_num() in ruby.h... yes
checking for rb_rational_den() in ruby.h... yes
checking for rb_array_const_ptr() in ruby.h... yes
checking for rb_sym2str() in ruby.h... yes
creating Makefile

current directory: /home/ruby/.rbenv/versions/2.3.8/lib/ruby/gems/2.3.0/gems/bigdecimal-1.4.0/ext/bigdecimal
make "DESTDIR=" clean
make: *** Нет правила для сборки цели «../../lib/bigdecimal.rb», требуемой для «Makefile».  Останов.

current directory: /home/ruby/.rbenv/versions/2.3.8/lib/ruby/gems/2.3.0/gems/bigdecimal-1.4.0/ext/bigdecimal
make "DESTDIR="
make: *** Нет правила для сборки цели «../../lib/bigdecimal.rb», требуемой для «Makefile».  Останов.

make failed, exit code 2

Gem files will remain installed in /home/ruby/.rbenv/versions/2.3.8/lib/ruby/gems/2.3.0/gems/bigdecimal-1.4.0 for inspection.
Results logged to /home/ruby/.rbenv/versions/2.3.8/lib/ruby/gems/2.3.0/extensions/x86_64-linux/2.3.0-static/bigdecimal-1.4.0/gem_make.out
```

**Comment by stomar** [comment]
Duplicate of #118, #119. Fixed by #120.

**Comment by mrkn** [comment]
I'm sorry for the inconvenience with my wrong packaging.
I've released the version 1.4.1 with #120.
Please retry with the new version.
